frontend/templates/troubleshooting/solution.tsx

```diff
@@ -215,7 +217,18 @@ export default function SolutionCard({
          <Flex gap="24px" direction="column">
             <SolutionHeader index={index} title={solution.heading} />
             <SolutionTexts body={solution.body} />
+            <LinkCards guides={solution.guides} />
          </Flex>
       </Flex>
    );
 }
+
+function LinkCards({ guides }: { guides: Guide[] }) {
+   return (
+      <Box>
+         {guides.map((guide: Guide) => (
+            <GuideResource key={guide.guideid} guide={guide} />
+         ))}
+      </Box>
+   );
+}
```

Comment:
Again, just allowing styling:
```suggestion
function LinkCards({ guides, ...props }: { guides: Guide[] } & BoxProps) {
   return (
      <Box {...props}>
         {guides.map((guide: Guide) => (
            <GuideResource key={guide.guideid} guide={guide} />
         ))}
      </Box>
   );
}
```